variables.yml

```diff
@@ -1,6 +1,18 @@
-release: '2.18.0'
-version: '2.18'
-release_date: 2021-06-01
+# PMM Version for HTML
+# See also mkdocs.yml plugins.with-pdf.cover_subtitle and output_path
+release: '2.19.0'
+version: '2.19'
+release_date: 2021-06-29
```

Comment:
@PaulJacobs-percona note to self: change to new release date when known (and see whether this variable is even used)

Reply:
Answer: `release_date` used in setting-up/server/virtual-appliance.md for OVA filename.